bundle_dao.go

```diff
@@ -66,8 +84,8 @@ func (d *bundleDAO) getLocales() ([]string, error) {
 	return lSlice, nil
 }
 
-func (d *bundleDAO) getComponentMessages(locale, component string) (ComponentMsgs, error) {
-	compDirPath := filepath.Join(d.cfg.LocalBundles, component)
+func (d *bundleDAO) GetComponentMessages(name, version, locale, component string) (ComponentMsgs, error) {

```

Comment:
why method return struct，not return the struct object point

Reply:
ComponentMsgs is an interface storing a pointer. So it returns a pointer in fact.

Reply:
Ok, I have found it